### DISCUSSION TITLE
Dear Github team! I found scammer who are using github on bad behavior.

### DISCUSSION THREAD

**Discussion by happyeagle912** [community/community#121686] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

https://github.com/miracle1125
This is fake gihub repository and owner is scammer.
Be careful!
Andres.

**Comment by ebndev** [comment]
Hi USERNAME, 👋🏻 We really appreciate you flagging this. The best route to get this to the proper GitHub team is to use our [abuse reporting tools](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam). Here's all the info:

- [Reporting abuse or spam](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam)
- [Reporting a user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-user)
- [Reporting an issue or pull request](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request)
- [Reporting a comment](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-comment)

You can report behavior and content that violates community guidelines and terms. We are going to close this post, but for this and any future incidents, please refer to the links above.

Thank you!
